Havit.Blazor.Components.Web.Bootstrap/Forms/Autosuggests/HxAutosuggest.cs

```diff
@@ -169,6 +175,7 @@ protected override void BuildRenderInput(RenderTreeBuilder builder)
 			builder.AddAttribute(1023, nameof(HxAutosuggestInternal<TItem, TValue>.InputGroupStartTemplate), this.InputGroupStartTemplate);
 			builder.AddAttribute(1024, nameof(HxAutosuggestInternal<TItem, TValue>.InputGroupEndText), this.InputGroupEndText);
 			builder.AddAttribute(1025, nameof(HxAutosuggestInternal<TItem, TValue>.InputGroupEndTemplate), this.InputGroupEndTemplate);
+			builder.AddAttribute(1026, nameof(HxAutosuggestInternal<TItem, TValue>.HighlightFirstSuggestion), this.HighlightFirstSuggestion);
```

Comment:
You have to pass `HighlightFirstSuggestionEffective` here.
The `HxAutosuggestInternal` parameter name should be named `HighlightFirstSuggestionEffective` as well.